pkgs/by-name/li/libguestfs/package.nix

```diff
@@ -47,6 +47,11 @@ assert appliance == null || lib.isDerivation appliance;
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "libguestfs";
+
+  # It is advisable to keep the `libguestfs` and `libguestfs-appliance-debian`
+  # packages in sync to prevent issues like https://github.com/NixOS/nixpkgs/issues/280881.
+  # In particular, this package should be updated only after the update in
+  # question lands in Debian testing (see `libguestfs-appliance-debian` package).
```

Comment:
We shouldn't rely on Debian and limit our updates based on them.

Reply:
We already rely on Debian as the source for the appliance in the first place. As most features of the libguestfs package depend on the appliance, it would make sense to me to keep them in sync for the sake of stability (see the linked issue).

Reply:
Actually nevermind, we might be able to use the `libguestfs-make-fixed-appliance` program from nixpkgs instead of debian. I'll investigate.

Reply:
Seems to work fine. Now the appliance package will update automatically with the `libguestfs` package, nice! (kind of stupid that I haven't used the nixpkgs version to begin with...)

Reply:
Ignore the last two comments, `libguestfs-make-fixed-appliance` is not in nixpkgs because supermin isn't.